radixdlt/src/test/java/com/radixdlt/ConsensusModuleTest.java

```diff
@@ -137,6 +141,27 @@ protected void configure() {
 				bind(BFTSyncTimeoutScheduler.class).toInstance(mock(BFTSyncTimeoutScheduler.class));
 				bind(BFTConfiguration.class).toInstance(bftConfiguration);
 				bind(Integer.class).annotatedWith(BFTSyncPatienceMillis.class).toInstance(200);
+
+				ECKeyPair ecKeyPair = ECKeyPair.generateNew();
+				bind(HashSigner.class).toInstance(ecKeyPair::sign);
+			}
+
+			@Provides
+			@PacemakerTimeout
+			long pacemakerTimeout() {
```

Comment:
As noted elsewhere would be better and more consistent if bound in configure like BFTSyncPatienceMillis